src/Raven.Server/ServerWide/Maintenance/ClusterMaintenanceSupervisor.cs

```diff
@@ -117,18 +116,19 @@ public class ClusterNode : IDisposable
             private DateTime _lastSuccessfulUpdateDateTime;
             private bool _isDisposed;
             private readonly string _readStatusUpdateDebugString;
-            private TcpConnectionInfo _tcpConnection;
+            private readonly string _apiKey;
 
             public ClusterNode(
                 string clusterTag,
                 string url,
-                TcpConnectionInfo tcpConnectionConnectionInfo,
+                string apiKey,
```

Comment:
Since we are talking to members of our own cluster, we don't need an API Key, we can generate the token directly